src/libraries/System.Security.Cryptography/src/Resources/Strings.resx

```diff
@@ -78,6 +78,15 @@
   <data name="Arg_InvalidType" xml:space="preserve">
     <value>Invalid type.</value>
   </data>
+  <data name="Argument_Asn1_InvalidCharacterString" xml:space="preserve">
+    <value>The specified UniversalTagNumber must be a string type.</value>
+  </data>
+  <data name="Argument_Asn1_InvalidStringContents" xml:space="preserve">
+    <value>The contents of the string are not valid for the '{0}' ASN.1 encoding.</value>
+  </data>
+  <data name="Argument_Asn1_InvalidDer" xml:space="preserve">
+    <value>The contents are not valid ASN.1 under Distinguished Encoding Rules.</value>
+  </data>
```

Comment:
Please help quiet my neuroses and move these collectively down one (Arg_RankMultiDimNotSupported should be above them, Argument_DestinationTooShort below)